src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -1703,9 +1703,22 @@ protected void AddRudeDeleteAroundActiveStatement(List<RudeEditDiagnostic> diagn
             Func<TSyntaxNode, TSyntaxNode, bool> areSimilar)
             where TSyntaxNode : SyntaxNode
         {
+            ReportUnmatchedStatements(diagnostics, match, new[] { syntaxKind }, oldActiveStatement, newActiveStatement, areEquivalent, areSimilar);
+        }
+
+            protected void ReportUnmatchedStatements<TSyntaxNode>(
```

Comment:
Thank you!

---
In reply to: [114823061](https://github.com/dotnet/roslyn/pull/19114#discussion_r114823061) [](ancestors = 114823061)